schemas/2015-04-01/Microsoft.NotificationHubs.json

```diff
@@ -1,268 +1,346 @@
-          ]
-        },
-        "properties": {
-          "type": "object",
-          "properties": {
-            "wnsCredential": {
-              "oneOf": [
-                { 
-                  "type": "object",
-                  "properties": {
-                    "packageSid": {
-                      "type": "string",
-                      "minLength": 1,
-                      "description": "Microsoft.NotificationHubs/namespaces/notificationHubs: The Package SID provided by Windows Live Application Management when registering the application."
-                    },
-                    "secretKey": {
-                      "type": "string",
-                      "minLength": 1,
-                      "description": "Microsoft.NotificationHubs/namespaces/notificationHubs: The Secret Key associated with the packageSid provided by Windows Live Application Management when registering the application."
-                    },
-                    "windowsLiveEndpoint": {
-                      "type": "string",
-                      "minLength": 1,
-                      "description": "Microsoft.NotificationHubs/namespaces/notificationHubs: The Windows Live Endpoint used to retrieve the Access Token for the application"
-                    }
-                  }
+                "apiVersion": {
+                    "enum": ["2014-09-01"]
                 },
-                { "$ref": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#/definitions/expression" }
-              ],
-              "description": "Microsoft.NotificationHubs/namespaces/notificationHubs: WNS credentials for a Notification Hub."
-            },
-            "apnsCredential": {
-              "oneOf": [
-                {
-                  "type": "object",
-                  "properties": {
-                    "apnsCertificate": {
-                      "type": "string",
-                      "minLength": 1,
-                      "description": "Microsoft.NotificationHubs/namespaces/notificationHubs: A certificate (in base 64 format) provided by Apple on the iOS Provisioning Portal"
-                    },
-                    "certificateKey": {
-                      "type": "string",
-                      "minLength": 1,
-                      "description": "Microsoft.NotificationHubs/namespaces/notificationHubs: The Certificate Key provided by the iOS Provisioning Portal when registering the application"
+                "properties": {
+                    "type": "object",
+                    "properties": {
+                        "name": {
+                            "type": "string",
+                            "minLength": 1,
+                            "description": "Microsoft.NotificationHubs/namespaces: The name of the namespace."
+                        },
+                        "namespaceType": {
+                            "oneOf": [{
+                                "enum": ["NotificationHub"]
```

Comment:
looks like the json style/format were changed. Please follow the patterns in this repo.